### ISSUE TITLE
Make nupnp discovery for Philips Hue optional at build-time

### ISSUE THREAD

**Issue by michielbdejong** [fxbox/foxbox#248]
Privacy-aware users should have an option to build their foxbox so that does not probe Philips's proprietary hosted service for nupnp.

**Comment by fabricedesre** [comment]
Can they prevent their Hue bridge from doing that?
Anyone can ping Philips's server, but that doesn't mean you have a Hue.

**Comment by michielbdejong** [comment]
We cannot stop the Hue Bridge from connecting to Philips's server, so a privacy-aware user should not use the Philips Hue Bridge at all.

We should just make sure that the privacy-aware build of foxbox does not ping Philips's nupnp server.

**Comment by fabricedesre** [comment]
Right. I'm fine with --something command line flag and each adapter could decide what to do based on its presence.
